_content/essays/spiritual-friendship.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Spiritual Friendship"
+drive_links:
+  - "https://drive.google.com/file/d/1rvXIqx1H_C-gxb9QVh3gszBK5DgvsH3E/view?usp=sharing"
+course: thai
+tags:
+  - friendship
```

Comment:
Yes! Friendship is a tag :)  Let's also add `bodhisatta` and `iddhi`  Also, make this `status: featured`